save-backend/src/main/kotlin/com/saveourtool/save/backend/service/OrganizationService.kt

```diff
@@ -43,25 +43,70 @@ class OrganizationService(
      * Mark organization with [organizationName] as deleted
      *
      * @param organizationName an [Organization]'s name to delete
+     * @param newStatus is new status for [organizationName]
      * @return deleted organization
      */
     @Suppress("UnsafeCallOnNullableType")
-    fun deleteOrganization(organizationName: String): Organization = getByName(organizationName)
+    fun changeOrganizationStatus(organizationName: String, newStatus: OrganizationStatus): Organization = getByName(organizationName)
```

Comment:
Why do we need to make a request to the database twice to get the `organization`? Let's pass the organization here, not the `organizationName`.